src/parser.rs

```diff
@@ -213,6 +213,27 @@ fn parse_data_type_definition(lexer: &mut RustyLexer, name: Option<String>) -> R
 
         Ok((DataTypeDeclaration::DataTypeDefinition { data_type : DataType::EnumType{ name, elements }}, None))
 
+    } else if lexer.token == Identifier && lexer.slice().eq_ignore_ascii_case("STRING") {   //STRING TODO: should STRING be a keyword?
```

Comment:
I think a keyword makes the most sense here.
I would have liked the datatypes to not be keywords, but i don't think this would work for strings
The only other idea i have is to figure out how ranged datatypes will be defined and see if this can be ported to strings, but I don't currently see how we can do that.

Reply:
I agree. Strings introduce a new gramatical concept not covered by other types (with the size-attachement to the datatype which is also not compatible of the grammar-understanding of an Array).

anyways ... its a keyword now